.github/workflows/ci.yml

```
@@ -86,6 +86,13 @@ jobs:
         if: ${{ success() }}
         run: ./ci/before_cache.sh
 
+      - name: Lint with stylua
```

nit: why mention the tool (implementation detail) in the step name ?

Also isn't this doing formatting, not linting?
```suggestion
      - name: Enforce code style
```

---

Happy to change the name, I put the name in the tool so it's more obvious which lint users should run locally, but we could add a `make format` command to automatically run our lua formatter of choice.

Re: lint vs. format, it's currently only checking the formatting in CI, it doesn't actually format the code (unless invoked locally by users without the --check flag)